huaweicloud/services/evs/resource_huaweicloud_evs_volume.go

```diff
@@ -184,17 +203,21 @@ func buildEvsVolumeCreateOpts(d *schema.ResourceData, config *config.Config) clo
 		volumeOpts.Metadata = m
 	}
```

Comment:
it seems that the metadata at least has one key.

Reply:
OFC